api/test/integration/run_tests.py

```diff
@@ -59,15 +59,17 @@ def collect_non_excluded_tests():
     return collected_tests
 
 
-def run_tests(collected_tests, n_iterations=1):
+def run_tests(collected_tests, n_iterations=1, html_report=False):
```

Comment:
I would put html_report to true by default

Reply:
In that case, the parameter would not be necessary. I'm removing it.